internal/frameworkprovider/service_resource.go

```diff
@@ -65,18 +64,61 @@ var serviceResourceSchema = func() schema.Schema {
 				Description: "Status of created service.",
 				AttributeTypes: map[string]attr.Type{
 					"slo_count": types.Int64Type,
-				},
-				PlanModifiers: []planmodifier.Object{
-					objectplanmodifier.UseStateForUnknown(),
+					"review_cycle": types.ObjectType{
+						AttrTypes: map[string]attr.Type{
+							"next": types.StringType,
+						},
+					},
 				},
 			},
 		},
 		Blocks: map[string]schema.Block{
-			"label": metadataLabelsBlock(),
+			"label":            metadataLabelsBlock(),
+			"responsible_user": serviceResponsibleUserBlock(),
+			"review_cycle":     serviceReviewCycleBlock(),
 		},
 	}
 }()
 
+func serviceReviewCycleBlock() schema.Block {
+	return schema.SingleNestedBlock{
+		Description: "Configuration for service review cycle.",
+		Attributes: map[string]schema.Attribute{
+			"rrule": schema.StringAttribute{
+				Required:    true,
+				Description: "Recurring rule in RFC 5545 RRULE format defining when a review should occur.",
+			},
+			"start_time": schema.StringAttribute{
+				Required: true,
+				MarkdownDescription: "Start time (inclusive) for the first occurrence defined by the rrule. " +
+					"Specify as an ISO 8601 date-time string without a time zone designator (e.g. 2024-01-02T15:04:05). " +
+					"The time zone is specified separately in the time_zone attribute.",
+				Description: "Start time for the first occurrence defined by the rrule, " +
+					"as an ISO 8601 date-time string without a time zone.",
+			},
+			"time_zone": schema.StringAttribute{
+				Required:            true,
+				MarkdownDescription: "Time zone identifier (IANA) used to interpret start_time and rrule times (e.g. Europe/Warsaw).",
+				Description:         "Time zone identifier used to interpret start_time and rrule times.",
+			},
+		},
+	}
+}
+
+func serviceResponsibleUserBlock() schema.Block {
```

Comment:
we don't want to be adding blocks to new resources anymore, we'll actually need to migrate from them some time in the future towards attributes, so let's change this to attribute list
there's only really one syntactic difference between blocks and attributes, attributes require `=` sign after the key name, while block don't have it